test/gtest/unit_conv_solver_ConvAsmImplicitGemmGTCDynamicBwdXdlopsNHWC.cpp

```diff
@@ -0,0 +1,164 @@
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+
+#include "unit_conv_solver.hpp"
+
+namespace {
+
+auto GetConvSmokeTestCases(miopenDataType_t datatype)
+{
+    using TestCase = miopen::unit_tests::ConvTestCase;
+
+    return std::vector{
+        // clang-format off
+        TestCase{{1, 8, 8, 8}, {8, 8, 3, 3}, {0, 0}, {1, 1}, {1, 1}, datatype},
+        // clang-format on
+    };
+}
+
+auto GetSmokeTestParams(miopenDataType_t datatype)
+{
+    Gpu supportedDevices = Gpu::gfx90A | Gpu::gfx94X;
+    if(datatype != miopenBFloat16)
+    {
+        supportedDevices = supportedDevices | Gpu::gfx908;
+    }
+    auto testParams = miopen::unit_tests::UnitTestConvSolverParams(supportedDevices);
+    testParams.Tunable(1);
+
+    return testParams;
+}
+
+auto GetConvFullTestCases(miopenDataType_t datatype)
+{
+    using TestCase = miopen::unit_tests::ConvTestCase;
+
+    return std::vector{
+        // clang-format off
+        TestCase{{16, 5, 225, 225}, {64, 5, 3, 3}, {0, 0}, {1, 1}, {1, 1}, datatype},
+        TestCase{{16, 576, 1, 1}, {576, 576, 1, 1}, {0, 0}, {1, 1}, {1, 1}, datatype},
+        TestCase{{16, 2048, 8, 32}, {4096, 2048, 1, 1}, {0, 0}, {1, 1}, {1, 1}, datatype},
+        TestCase{{16, 2048, 8, 32}, {2048, 2048, 1, 1}, {0, 0}, {1, 1}, {1, 1}, datatype},
+        TestCase{{16, 2048, 48, 32}, {2048, 2048, 3, 1}, {0, 0}, {3, 1}, {1, 1}, datatype},
+        TestCase{{16, 2048, 1, 512}, {2048, 2048, 1, 2}, {0, 0}, {1, 2}, {1, 1}, datatype},
+        TestCase{{16, 2048, 1, 30}, {576, 2048, 1, 1}, {0, 0}, {1, 1}, {1, 1}, datatype},
+        // clang-format on
+    };
+}
+
+auto GetFullTestParams(miopenDataType_t datatype)
+{
+    Gpu supportedDevices = Gpu::gfx90A | Gpu::gfx94X;
+    if(datatype != miopenBFloat16)
+    {
+        supportedDevices = supportedDevices | Gpu::gfx908;
+    }
+    auto testParams = miopen::unit_tests::UnitTestConvSolverParams(supportedDevices);
+    testParams.Tunable(1000);
+
+    return testParams;
+}
+
+} // namespace
+
+using GPU_UnitTestConvSolverConvAsmImplicitGemmGTCDynamicBwdXdlopsNHWCBwd_FP16 =
+    GPU_UnitTestConvSolverBwd_FP16;
+using GPU_UnitTestConvSolverConvAsmImplicitGemmGTCDynamicBwdXdlopsNHWCBwd_BFP16 =
+    GPU_UnitTestConvSolverBwd_BFP16;
+using GPU_UnitTestConvSolverConvAsmImplicitGemmGTCDynamicBwdXdlopsNHWCBwd_FP32 =
+    GPU_UnitTestConvSolverBwd_FP32;
+using CPU_UnitTestConvSolverConvAsmImplicitGemmGTCDynamicBwdXdlopsNHWCDevApplicabilityBwd_NONE =
+    CPU_UnitTestConvSolverDevApplicabilityBwd_NONE;
```

Comment:
More information about naming is [here](https://github.com/ROCm/MIOpen/pull/3446#pullrequestreview-2520943560).
```suggestion
using GPU_UnitTestConvSolverAsmImplicitGemmGTCDynamicBwdXdlopsNHWCBwd_FP16 =
    GPU_UnitTestConvSolverBwd_FP16;
using GPU_UnitTestConvSolverAsmImplicitGemmGTCDynamicBwdXdlopsNHWCBwd_BFP16 =
    GPU_UnitTestConvSolverBwd_BFP16;
using GPU_UnitTestConvSolverAsmImplicitGemmGTCDynamicBwdXdlopsNHWCBwd_FP32 =
    GPU_UnitTestConvSolverBwd_FP32;
using CPU_UnitTestConvSolverAsmImplicitGemmGTCDynamicBwdXdlopsNHWCDevApplicabilityBwd_NONE =
    CPU_UnitTestConvSolverDevApplicabilityBwd_NONE;
```